### PR TITLE
More updates for qwicklabs

### PR DESCRIPTION
**Pull request by chrislovecnm**

### CODE REVIEW

File: gke-to-gke-peering/README-QWICKLABS.md [hunk 1]
```diff
@@ -105,53 +102,23 @@ those clusters.
...
-some resources will most likely need to be increased. The Setup section below covers
-increasing the quotas.
+For qwicklabs you do not have to make these changes. These changes can be made if you are more advanced.
 
-Access to an existing Google Cloud project with the Kubernetes Engine service enabled
-If you do not have a Google Cloud account please signup for a free trial
-[here](https://cloud.google.com).
+1. Region for subnets and Node CIDR can be customized in /network/network.yaml.
+1. Cluster attributes like zone, image, node count, cluster CIDR and service CIDR can be customized in clusters/cluster.yaml.
+1. To add additional custom attributes to network or clusters yaml files and deployment manager scripts at /network/*.py or clusters/*.py needs to be updated accordingly.
```
Comment (raddaoui) [inline gke-to-gke-peering/README-QWICKLABS.md]: "star" is a special character, you need to escape it \\*.py. also sentence is not very clear. Maybe add a comma after deployment manager.

File: gke-to-gke-peering/README-QWICKLABS.md [hunk 2]
```diff
@@ -201,14 +167,14 @@ from service-ipv4-cidr.
...
 
-
 ## Troubleshooting
 
 1. Remember to enable API's as mentioned in deployment steps in the project where the resources are to be created. Otherwise, API not enabled error is thrown.
 1. Verify that the project is associated with a valid billing account.
 1. Make sure to have the right permissions for the GCP account to create above GCP/Kubernetes Engine resources in project. Otherwise, permission denied error is thrown.
 1. Make sure that the deployments created through install script are deleted before you try to re-install the resources. Otherwise, resources will not be installed properly.
 1. If there are any errors in cleanup script execution, refer to steps for deleting resources manually.
+1. If you get an error while running the install script run the cleanup script or manually delete the resources and attempt the install script again.
```
Comment (raddaoui) [inline gke-to-gke-peering/README-QWICKLABS.md]: Comma missing here too: install script, run the cleanup

**Comment by chrislovecnm** [comment]
Merging, cause I have to. @raddaoui fixed your comment.

**Pull request merged**